deploy/environments/applications.yaml

```
@@ -153,4 +155,4 @@ spack:
     - touchdetector
     - ultraliser
     - unit-test-translator
-    # - zee
+    # - zee
```

No newline at end of file?
```suggestion
    # - zee
```

---

@matz-e Thank you for your corrections !

---

@Nabil-AL you're welcome! Please note that I've also upgraded @BlueBrain/dke 's permissions, you should be able to push branches here and merge pull requests (we let you merge the PRs you open after approval)

---

Thank you, by the way I cannot commit your suggestion as github considers that no changes were made to the code

---

Sigh, fair enough. The next edit to the file will probably restore that line ending.